pandas/tests/indexes/test_interval.py

```diff
@@ -491,6 +594,188 @@ def testcontains(self):
         assert not i.contains(20)
         assert not i.contains(-20)
 
+
+    @pytest.mark.xfail(reason="new indexing tests for issue 16316")
```

Comment:
`Interval.covers()` isn't an index method (unlike `IntervalIndex.covers()`), so it belongs in the test file for `Interval`, not `IntervalIndex`.